### ISSUE TITLE
Crops of lead images

### ISSUE THREAD

**Issue by skleinfeldt** [jazkarta/jazkarta.com#39]
Lead images on Projects (and potentially future content types like blog posts idea) serve 2 purposes. When looking at the Project, the full image should be shown in a large size. When looking at a listing of Projects - in the bottom slider, or on the isotope/mixitup filtering page - a crop of the full image is needed. The crop will be a standard aspect ratio (to make the slider/filtering look nice) and it will be a close up of a portion of the screenshot that displays something (usually header logo) that identifies the project (so we can hid the titles on the slider).

In other words, I need to use the image cropping tool on lead images. Is this possible?

**Comment by skleinfeldt** [comment]
@alecpm I hear from the others that you are the person with the answer to this question - they thought maybe the tool works on lead images so it should be fine. If that is not the case the likely solution is to add another image field to hold a pre-cropped image. Please let me know.

**Comment by alecpm** [comment]
As discussed p.a.imagecropping allows you to set a specific (fixed aspect) crop for each image scale, it should work perfectly for this usecase.
